modules/oci/services/grafana.scm

```diff
@@ -320,6 +321,15 @@ to \"host\" the @code{port} field will be ignored."))
               (if (and password-file (maybe-value-set? password-file))
                   '(sops-secrets)
                   '()))
+             (container-user (if (eq? 'podman runtime)
+                                 uid
+                                 ;; FIXME: is the empty string the same as a
+                                 ;; non-initialized `maybe-string'?
```

Comment:
No, maybe values are set by default to a special `%unset-value` variable. I would avoid setting that explicitly in case it is changed in the future in Guix mainline. I think you can achieve to have the default value set with the Docker runtime by doing something similar to this https://github.com/fishinthecalculator/gocix/blob/main/modules/oci/services/grafana.scm#L334 . I think it should look like this:

```
(if (eq? 'podman runtime)
     (mainline:oci-container-configuration
       (inherit container-config)
       (container-user uid))
     container-config)
```